tests/e2e/util.go

```diff
@@ -916,19 +916,18 @@ func getSvcClientAndNamespace() (clientset.Interface, string) {
 // FULL_SYNC_INTERVAL_MINUTES in deployment template. For this to take effect,
 // we need to terminate the running csi controller pod.
 // Returns fsync interval value before the change.
-func updateCSIDeploymentTemplateFullSyncInterval(client clientset.Interface, mins string, namespace string) string {
+func updateCSIDeploymentTemplateFullSyncInterval(client clientset.Interface, mins string, namespace string) error {
```

Comment:
since you are changing the function signature, better to a take a look at all the places it is used as well

Reply:
This function is being used only in test "Verify CNS volume is synced with updated GC PV & PVC labels after SVC connection is restored" for GC, which I have already tested.